src/main/java/lotto/domain/LottoWinning.java

```diff
@@ -1,37 +1,79 @@
 package lotto.domain;
 
+import lotto.domain.value.LottoNumber;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.util.EnumMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 
 public class LottoWinning {
 
     private static final int LOTTO_SIZE = 6;
+
     private static final String FORM_ERROR_MSG = "당첨번호는 6자리 입니다!!!!";
+    private static final String NUMBER_CHECK_ERROR_MSG = "숫자만 입력 가능합니다!!!!";
+    private static final String RATE_PATTERN = "0.##";
 
     private final Lotto winningNumbers;
+    private final LottoNumber bonusBall;
 
-    private LottoWinning(List<Integer> winningNumbers) {
+    private LottoWinning(List<Integer> winningNumbers, int bonusBall) {
 
         this.winningNumbers = Lotto.winningFrom(winningNumbers);
+        this.bonusBall = new LottoNumber(bonusBall);
 
     }
 
-    public static LottoWinning from(List<Integer> winningNumbers) {
+    public static LottoWinning from(List<Integer> winningNumbers, String bonusBall) {
 
         if(winningNumbers.size() != LOTTO_SIZE) {
             throw new IllegalArgumentException(FORM_ERROR_MSG);
         }
+        constantCheck(bonusBall);
+        if (winningNumbers.contains(bonusBall)) {
+            throw new IllegalArgumentException("당첨번호에 있는 번호는 보너스 번호가 아닙니다!!!");
+        }
 
-        return new LottoWinning(winningNumbers);
+        return new LottoWinning(winningNumbers, Integer.parseInt(bonusBall));
     }
 
     public Map<Rank, Integer> createRepository(LottoTicket lottoTicket) {
-        return lottoTicket.createRepository(winningNumbers);
+
+        Map<Rank, Integer> repository = new EnumMap(Rank.class);
```

Comment:
```suggestion
        Map<Rank, Integer> repository = new EnumMap<>(Rank.class);
```

다이아몬드 표현같은건 생략하지 말아주세요 ㅎ